### PR TITLE
Updating test_element to correctly test for 'checked' property

### PR DESCRIPTION
**Pull request by jimevans**

### CODE REVIEW

**Comment by jimevans** [comment]
I've changed the test to (1) correctly verify there is no "checked" attribute on the element, and to (2) correctly retrieve the "checked" property of the element.

File: webdriver/tests/state/get_element_property.py [hunk 1]
```diff
@@ -161,9 +161,9 @@ def test_element(session):
     session.url = inline("<input type=checkbox>")
     element = session.find.css("input", all=False)
     element.click()
-    assert None == session.execute_script("return arguments[0].hasAttribute('checked')",
-                                          args=[element])
-    result = session.transport.send("GET", "session/{session_id}/element/{element_id}/property/id"
+    assert False == session.execute_script("return arguments[0].hasAttribute('checked')",
+                                           args=[element])
```
Comment (andreastt) [inline webdriver/tests/state/get_element_property.py]: This is fine, but do you mean also replacing `==` with `is`, and converting the args list to a tuple? This statement should read:

```py
assert session.execute_script("return arguments[0].hasAttribute('checked')", args=(element,)) is False
```

**Review by andreastt — CHANGES_REQUESTED** [review]
This is fine, but I wonder if you can make one additional change whilst you’re at it.

File: webdriver/tests/state/get_element_property.py (hunk 1 above)
Comment (jimevans) [reply webdriver/tests/state/get_element_property.py]: Gladly. Done in next commit.

**Review by andreastt — APPROVED** [review]

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24756)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/344860986)

**Pull request merged**